integration/tests/assertions.go

```diff
@@ -33,3 +39,42 @@ func ListNodes(clientset kubernetes.Interface, nodeGroupName string) *corev1.Nod
 	Expect(err).NotTo(HaveOccurred())
 	return nodeList
 }
+
+func AssertNodeVolumes(kubeConfig string, region string, nodeGroupName string, volumeName string) {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeConfig)
+	Expect(err).NotTo(HaveOccurred())
+	clientSet, err := kubernetes.NewForConfig(config)
+	Expect(err).NotTo(HaveOccurred())
+	nodes, err := clientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	var instanceIDs []string
+	for _, node := range nodes.Items {
+		// only test instances in this nodegroup, any other should be ignored.
+		if node.Labels[api.NodeGroupNameLabel] != nodeGroupName {
+			continue
+		}
```

Comment:
Based on the above suggestion, this can be removed.
```suggestion
```

Reply:
Ah that's true, nice. :)